pkg/native/serializers/beta/serializer_spdx3.go

```diff
@@ -102,9 +108,9 @@ type hashList struct {
 	HashValue string `json:"hashValue"`
 }
 
-type SPDX3 struct{}
+type SerializerSPDX3 struct{}
```

Comment:
@puerco I streamlined it to other serializer naming

Reply:
Lol, I was actually trying to start destuddering all the serializers and unserializers. Since the serializers now live in a `serializers` packag, I thought it would be better to have:

`serializers.SPDX3` instead of `serializers.SerializerSPDX3`.

I don't care a lot about names and we can change them if they feel more natural.

Reply:
I can do us a favor and rename all existing Serializers to shorted names. 
I prefer the shorter version, just thought we should be consistent

Reply:
serializers.SPDX3
serializers.SPDX23
serializers.CDX
...

Reply:
Done :)